ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.18-nightly-x86.yaml

```diff
@@ -173,8 +173,8 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-azure
   timeout: 8h0m0s
-- always_run: false
-  as: control-plane-multi-120nodes
+- as: control-plane-multi-120nodes
+  cron: 0 2 4,18 * *
```

Comment:
hour 10 is better

suggest: '0 10 4,18 * *'

Reply:
Thanks. Updated.